### PR TITLE
Add color back

### PR DESCRIPTION
**Pull request by AlbanAndrieu**
Add color in Nomad UI 
image

### CODE REVIEW

File: tasks/host_volume.yml [hunk 1]
```diff
@@ -7,3 +7,4 @@
     state: "{{ item['state'] | default('directory') }}"
     mode: "{{ item['mode'] | default('0755') }}"
   with_items: "{{ nomad_host_volumes }}"
+  ignore_errors: true
```
Comment (rndmh3ro) [inline tasks/host_volume.yml:10]: Is there a reason this and the config in the client.hcl.j2 is added here?

File: templates/base.hcl.j2 [hunk 2]
```diff
@@ -130,3 +130,14 @@ autopilot {
     server_stabilization_time = "{{ nomad_autopilot_server_stabilization_time }}"
 }
 {% endif %}
+
+{% if nomad_ui | default(False) | bool == True %}
```
Comment (rndmh3ro) [inline templates/base.hcl.j2:134]: Can you please document the following vars in the readme?
- nomad_ui
- nomad_ui_label_test
- nomad_ui_label_background_color
- nomad_ui_label_text_color

They also need a default value.

**Review by rndmh3ro — CHANGES_REQUESTED** [review]

File: templates/base.hcl.j2 (hunk 2 above)
Comment (AlbanAndrieu) [reply templates/base.hcl.j2:134]: Thanks for you feedback @rndmh3ro 
I added the doc in the README and I also fix spelling mistake, nomad_ui_label_test to nomad_ui_label_text

File: tasks/host_volume.yml (hunk 1 above)
Comment (AlbanAndrieu) [reply tasks/host_volume.yml]: I removed it

File: templates/base.hcl.j2 (hunk 2 above)
Comment (rndmh3ro) [reply templates/base.hcl.j2:135]: Thanks!

We still need a default value for theses variables.
So if we set `nomad_ui` to `true`, but not set `nomad_ui_label_text`, the run will fail.

Comment (AlbanAndrieu) [reply templates/base.hcl.j2:135]: > Thanks!
> 
> We still need a default value for theses variables. So if we set `nomad_ui` to `true`, but not set `nomad_ui_label_text`, the run will fail.

Your wishes are my command

Comment (rndmh3ro) [reply templates/base.hcl.j2:135]: Sorry, my request wasn't clear enough.

The defaults hould be set in the `defaults/main`.yml

Comment (AlbanAndrieu) [reply templates/base.hcl.j2:135]: Fine. I did it

File: templates/client.hcl.j2 [hunk 3]
```diff
@@ -96,6 +96,13 @@ client {
     template {
     {{ template_config(nomad_template_config) | replace('\'', '\"') }}
     }
+
+    {% if nomad_artifact -%}
+    artifact {
+    {% for key, value in nomad_artifact.items() %}
+    "{{ key }}" = "{{ value }}"
+    {% endfor -%}
+    }
```
Comment (rndmh3ro) [inline templates/client.hcl.j2:105]: Here an `endif` seems to be missing:

```suggestion
    {% if nomad_artifact -%}
    artifact {
    {% for key, value in nomad_artifact.items() %}
    "{{ key }}" = "{{ value }}"
    {% endfor -%}
    {% endif %}
    }
```

**Review by rndmh3ro — APPROVED** [review]
Thanks, LGTM! Sorry for the back and forth in this PR.

**Comment by AlbanAndrieu** [comment]
I appreciate you taking the time to look over my MR, @rndmh3ro.
I'm glad to give back to the community.

**Pull request merged**